docs/disable_resources.md

```diff
@@ -0,0 +1,120 @@
+
+Logs will remain visible for disabled resources, along with an “Enable Resource” button in the upper left of the long pane.
+
+<figure>
+  <img src="/assets/img/disable-resources-detail-view.png" alt="A screenshot of the Tilt Web UI's Detail View with a disabled resource selected. The central log pane shows all the resource's logs.">
+</figure>
+
+### From the Tilt API
+You can also enable and disable resources programmatically.
+
+The source of truth for a resource’s disabled status is a [ConfigMap](https://api.tilt.dev/core/config-map-v1alpha1.html). To enable or disable a specific resource, you’ll need to edit the ConfigMap.
+
+Using Tilt’s sample [pixeltilt project](https://github.com/tilt-dev/pixeltilt/), let’s walk through the steps of disabling a specific resource called `glitch`, accessing the Tilt API via the `tilt` command.
+
+First, find the name of the ConfigMap that corresponds to the resource we want to disable. (Tip: Disable ConfigMap names generally follow the pattern of `<uiresource name>-disable`.)
+
+Start by listing all resources to find the [UIResource](https://api.tilt.dev/interface/ui-resource-v1alpha1.html) name:
+```shell
+$ tilt get uiresource
+
+NAME              CREATED AT
+storage           2021-12-23T16:19:51Z
+muxer             2021-12-23T16:19:51Z
+object-detector   2021-12-23T16:19:51Z
+frontend          2021-12-23T16:19:51Z
+glitch            2021-12-23T16:19:51Z
+color             2021-12-23T16:19:51Z
+bounding-box      2021-12-23T16:19:51Z
+(Tiltfile)        2021-12-23T16:19:51Z
+```
+
+Then look at the details of the specific resource to find the ConfigMap name:
+``` shell
+$ tilt get uiresource glitch -ojson
+
+{
+    "apiVersion": "tilt.dev/v1alpha1",
+    "kind": "UIResource",
+    …
+    "status": {
+        "disableStatus": {
+            "disabledCount": 1,
+            "enabledCount": 0,
+            "sources": [
+                {
+                "configMap": {
+                    "key": "isDisabled",
+                    "name": "glitch-disable"
+                    }
+                }
+            ]
+        }
+    }
+}
+```
+The configmap name will be under the UIResource’s `status.disableStatus.sources[0].configMap.name`.
+
+Now that you have the ConfigMap name, edit it with the desired disabled state:
+
+```shell
+$ tilt patch configmap glitch-disable -p '{ "data": { "isDisabled": "true" } }'
+```
+
+## What can I use this feature for? 
+We saw that for many cases, it’s not practical or feasible to run all your resources all the time. But changing resources can interrupt your flow (e.g., you have to switch between the UI and command line). Improving Tilt's interface can make it more seamless to change the resources you’re running at any given time, and reduce the cognitive load of finding and running the right resources you need to run.
+
+So, we’re planning these future features to make it easier to manage sets of resources, and pick up from where you left off every time you restart Tilt:
```

Comment:
This part doesn't seem to really match the title "What can I use this feature for". It seems like this section is really two different sections: "Why do I want to use this" and "How will I be able to use this in the future". Maybe add the latter as a smaller, sub title?